TEST_RUN_EVENTS.csv

```diff
@@ -0,0 +1,39 @@
+Test Run Events,,,
+,,,
+EVENT_SUFFIX,Description,,
+_INFO,Adding extra context for an specific task,,
+_SUCCESS,The task has finalized with no errors,,
+_ERROR,An execution problem was found,,
+_START,A new task section has begun,,
+,,,
+,,,
+,,,
+,,,
+,,,
+Stage  ,Event Type,Description,Definition
+Trigger,CREATED_INFO,Trigger Run has been created,
+Trigger,RESOLVE_ERROR,Resolving trigger details failed,
+Trigger,RESOLVE_SUCCESS,Resolving trigger details failed,
+Trigger,RESOLVE_START,Resolving trigger details based on environment variables,
+Trigger,EXECUTION_START,Initial trigger execution,
+Trigger,EXECUTION_SUCCESS,Successful trigger execution,
+Trigger,HTTP_UNREACHABLE_HOST_ERROR,Tracetest could not reach the defined host in the trigger,
+Trigger,DOCKER_COMPOSE_HOST_MISMATCH_ERROR,"We identified Tracetest is running in docker compose, to connect to the host machine use the `host.docker.internal` hostname. For more information see https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds",

```

Comment:
Trigger,DOCKER_COMPOSE_HOST_MISMATCH_ERROR,"We identified Tracetest is running in docker compose, so if trying to access your local host machine please use the `host.docker.internal` hostname. For more information see https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds",